app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalController.php

```diff
@@ -84,4 +85,32 @@ public function show(): JsonResponse
             'footer_logo' => Vite::asset('resources/svg/CGBS_Logo_FullColor_Light.svg'),
         ]);
     }
+
+    public function getProductLicenses(): JsonResponse
+    {
+        $licenses = auth('contact')->user()->productLicenses()
+            ->with(['product:id,name,version'])
+            ->get()
+            ->map(function ($license) {
+                $license->formatted_expiration_date = $license->expiration_date
+                    ? $license->expiration_date->format('m-d-Y')
+                    : null;
```

Comment:
Not sure, but this might be a more succinct way of writing this.

```suggestion
                $license->formatted_expiration_date = $license->expiration_date?->format('m-d-Y');
```